template/[% if repo_platform == 'github' %].github[% endif %]/workflows/ci.yml.jinja

```diff
@@ -98,11 +109,43 @@ jobs:
       - run: git status --porcelain
       - run: test -z "$(git status --porcelain)"
 [%- endif %]
-on:
-  pull_request:
-    types:
-      - opened
-      - synchronize
-  push:
-    branches:
-      - main
+  semantic-release:
+    name: Semantic Release
+    runs-on: ubuntu-22.04
+    # Avoid semantic release for forked repository
+    if: github.event_name != 'pull_request' && github.repository == 'serious-scaffold/ss-python'
+    permissions:
+      contents: write
+      id-token: write
+      issues: write
+      pull-requests: write
+    needs: ci
```

Comment:
```suggestion
    needs: [ci
[%- if project_name == "Serious Scaffold Python" -%]
    , consistency
[%- endif -%]
    ]
```

Reply:
release job needs consistency for ss-python.

Reply:
I changed to multiline list, thanks for pointing out.